src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -88,6 +88,21 @@ public void GetApiDescription_PopulatesGroupName()
             Assert.Equal("Customers", description.GroupName);
         }
 
+        [Fact]
+        public void GetApiDescription_PopulatesGroupName_FromMetadata()
```

Comment:
Could you add an additional unit tests to exercise the fallback path (when both EndpointGroupName and GroupName are specified)?